scripts/asciidoc2confluence.groovy

```diff
@@ -57,13 +57,8 @@ def baseUrl
 
 // configuration
 def config
-try {
-    println "scriptBasePath: ${scriptBasePath}"
```

Comment:
I don't know in which case `scriptBasePath` would be available if run from build.gradle, please clarify if that's needed for other reasons.

Reply:
afaik, the scriptBasePath was introduced for the stand-alone version of asciidoc2confluence